src/routes/contact-us/+page.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+	import fb_logo from './Facebook_Logo_Primary.png';
+	import ig_logo from './Instagram_Glyph_Gradient.png';
+</script>
+
+<h1>Contact Us</h1>
+
+<p>
+	Email Address: <a href="mailto:kendoinfo@umich.edu"> kendoinfo@umich.edu</a>
+</p>
+
+<h2>Find us on:</h2>
+
+<figure>
+	<a href="https://www.facebook.com/groups/1038072369705184">
+		<img class="social-media" src={fb_logo} alt="Facebook logo" />
+	</a>
+	<a href="https://www.instagram.com/kendoumich/">
+		<img class="social-media" src={ig_logo} alt="Instagram logo" />
+	</a>
+</figure>
+
+<style>
+	.social-media {
+		padding: 1em;
+		width: 100px;
+		height: auto;
+	}
+
+	.social-media:hover {
```

Comment:
This feels a bit strange to me.

Reply:
What feels strange? The whitespace on line 29?

Reply:
No just like using the website and having the logo fade out when you hover over it.

Reply:
I did that bc it was in one of the [W3 Schools Tutorials](https://www.w3schools.com/howto/howto_css_social_media_buttons.asp#:~:text=.fa%3Ahover%20%7B%0A%C2%A0%20opacity%3A%200.7%3B%0A%7D) and it seemed reasonable to have something change about the logo when it was hovered. Is there a better thing to do for hover?

Reply:
Firstly, be careful with W3 School, they are beginner friendly but they are often outdated (just look at the logos in the example you linked) and sometimes they leave out important information in order to remain simple.

The effect sort of works in the example because they don't have dark mode so making the image more transparent makes it more white and brighter is kind of like an active state. This was especially popular a while ago when this article was most likely written.

You could try a number of things and see what works. Try things like brighter, bigger, more saturated, outlined, or elevated these are positive attention drawing features. Reducing opacity makes it faded which is the opposite of what you want. (You could try reducing the opacity whenever it's not hovered but I'm not really a fan of this). You should visit some other sites and see how they do it. Lots of people have social media links on their websites.

You should also always have a focus effect whenever you have a hover effect for the same reason you should always have a enter key listener whenever you have a click listener.

Reply:
After conversation, we decided to relocate/move some of the social media. Resolved in [5e58e97](https://github.com/kendoclub-at-umich/website/pull/32/commits/5e58e970d6d50441610fd35db7b9caf3c7f235e7)